RANSACApp/main.py

```diff
@@ -27,198 +27,207 @@
 SHOW_COLOR_IMAGE_NAME = "-SHOWCOLORIMAGE-"
 
 def main():
-  in_roi_mode = False
-
-  # Get Configuration
-  config: RansacConfig = RansacConfig.load()
-  config.save()
-
-  roi_layout = [
-                [sg.Graph((640, 480), (0, 480), (640, 0), key=ROI_SELECTION_NAME,drag_submits=True, enable_events=True)]
-               ]
-
-  # Define the window's contents
-  tracking_layout = [
-                     [sg.Text("Threshold"), sg.Slider(range=(0, 100), default_value=config.threshold, orientation = 'h', key=THRESHOLD_SLIDER_NAME)],
```

Comment:
I had this lined up this way because it reflected how the layout actually ends up in the GUI. That said, if it's gonna make people's formatters freak out, I guess I can live.